cmd/zed/zed.d/zed-functions.sh

```diff
@@ -511,10 +511,7 @@ zed_guid_to_pool()
 		return
 	fi
 
-	guid=$(printf "%llu" "$1")
-	if [ -n "$guid" ] ; then
-		$ZPOOL get -H -ovalue,name guid | awk '$1=='"$guid"' {print $2}'
-	fi
+	$ZPOOL get -H -ovalue,name guid | awk '$1 == '"$1"' {print $2; exit}'
```

Comment:
Dropping the `printf` would mean this function no longer accepts the GUID as either decimal or hex.  We should keep that functionality even though the only callers we provide all pass `$ZEVENT_POOL_GUID`.  Others may have written their own scripts which depend on this behavior.

Reply:
urgh, this worked in my tests because GAWK supports hex integers as an extension, but MAWK (and, presumably, POSIX AWK) doesn't seem to

updated to retain the printf

Reply:
> updated to retain the printf

I don't see the printf

Reply:
odd, coulda swore i added it back when i made the comment